ink.scribl.Scribl.yml

```diff
@@ -0,0 +1,34 @@
+app-id: ink.scribl.Scribl
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: scribl
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  # Permission to write to app-specific configuration and cache directories.
+  - --filesystem=xdg-config
+  - --filesystem=xdg-cache
+  - --filesystem=xdg-data
+
+modules:
+  - name: scribl
+    sources:
+      - type: git
+        url: https://github.com/jneem/scribl
+        branch: trunk
+    buildsystem: simple
+    build-commands:
+      - source /usr/lib/sdk/rust-stable/enable.sh && cargo build --release
+      - install -D target/release/scribl /app/bin/scribl
+      - install -D metadata/ink.scribl.Scribl.appdata.xml /app/share/metainfo/ink.scribl.Scribl.appdata.xml
+      - install -D metadata/ink.scribl.Scribl.svg /app/share/icons/scalable/apps/ink.scribl.Scribl.svg
+      - install -D metadata/ink.scribl.Scribl.desktop /app/share/applicatons/ink.scribl.Scribl.desktop
+    build-options:
+      build-args:
+        - --share=network
```

Comment:
The builders don't have network access during the build stage. The sources must be downloaded before; please use https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo